packages/sitecore-jss-cli/src/create/index.ts

```diff
@@ -125,9 +127,41 @@ export function getPascalCaseName(name: string): string {
  * @param {string} projectFolder Project folder
  * @param {string} name Name value to replace
  * @param {string} prefix Prefix of the sample app's template - should match Jss[RAV|Next]Web
+ * @param {string} withPrefix if true, replaces prefix with app name in PascalCase,
+ * otherwise strip the prefix
  */
-export function replacePrefix(projectFolder: string, name: string, prefix: string) {
-  console.log(chalk.cyan(`Replacing template prefix with ${getPascalCaseName(name)}`));
+export function replacePrefix(
+  projectFolder: string,
+  name: string,
+  prefix: string,
+  withPrefix?: string
+) {
+  const value: boolean = withPrefix?.toLowerCase() === 'true' ? true : false;
+
+  if (!value) {
+    console.log(chalk.cyan('Removing template prefix...'));
+    name = '';
```

Comment:
I'd avoid repurposing the `name` param in this case (usually bad practice to modify incoming parameters), and just use empty string directly below.